tests/SessionTest.php

```diff
@@ -7,7 +7,7 @@
 
 class SessionTest extends TestCase
 {
-    public function setUp()
+    protected function setUp()
```

Comment:
According to the [PHPUnit fixtures reference](https://phpunit.readthedocs.io/en/latest/fixtures.html?highlight=fixtures), the `setUp` method is protected, not public.